packages/menu/src/Menu.ts

```diff
@@ -138,12 +138,18 @@ export class Menu extends SpectrumElement {
         if (code !== 'ArrowDown' && code !== 'ArrowUp') {
             return;
         }
-        event.preventDefault();
+        const lastFocusedItem = this.menuItems[this.focusedItemIndex];
         const direction = code === 'ArrowDown' ? 1 : -1;
-        this.focusMenuItemByOffset(direction);
+        const itemToFocus = this.focusMenuItemByOffset(direction);
+        console.log(itemToFocus, lastFocusedItem);
```

Comment:
Will assume lint checks will catch this console.log. :)

Reply:
`eslint-plugin-lucier` was a great add to the project 😬 

I'll look at getting something official in here soon!